.github/workflows/cherryPick.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           token: ${{ secrets.OS_BOTIFY_TOKEN }}
 
       - name: Set up git for OSBotify
+        id: setupGitForOSBotify
         uses: Expensify/App/.github/actions/composite/setupGitForOSBotifyApp@8c19d6da4a3d7ce3b15c9cd89a802187d208ecab
```

Comment:
Not introduced in this PR but I think either back to main or a relative path (i.e: code from the same ref that triggered the workflow) would be good

Reply:
Over a specific commit SHA? From the docs

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#runsstepsuses

>Using the commit SHA of a released action version is the safest for stability and security.